bootstrap.py

```diff
@@ -52,6 +53,7 @@ def get_architecture():
 parser.add_option("-f", "--force", dest="force", action="store_true", help="Force registration (will erase old katello and puppet certs)")
 parser.add_option("-r", "--release", dest="release", default=RELEASE, help="Specify release version")
 parser.add_option("-R", "--remove-rhn-packages", dest="removepkgs", action="store_true", help="Remove old Red Hat Network Packages")
+parser.add_option("-U", "--unmanaged", dest="unmanaged", action="store_true", help="Add the server as unmanaged. Useful to skip provisioning dependencies.")
```

Comment:
I'd not add a shortopt, IMHO shortopts should be reserved for "important" options, and having -u and -U is confusing (not that the current code would not be already guilty of that)